File: webapi/Models/Pessoa.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;

namespace WebApi.Models;

//USER
public class Pessoa
{
    
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public String ?Id{get; set;}
    
    [Required]
    public String FirstName {get; set;}
    
    public String ?Surname {get; set;}
    
    [Required]
    public int Age{get; set;}
    
    public DateTime? CreationDate{get; set;}
}

File: cadastropessoa/lib/Pessoa.dart
class PessoaBody{
  late String firstName;
  late String surname;
  late int age;

  set firstname(String firstname) {
    firstName = firstname;
  }
  
  Pessoa(){
    
  }

  Map<String, dynamic>toJson(){
    return {
      "firstname":firstName,
      "surname":surname,
      "age":age
    };
  }
}